webui/react/src/shared/bin/test.py

```diff
@@ -7,7 +7,7 @@
 parser = argparse.ArgumentParser()
 
 SHARED_WEB_REPO='https://github.com/determined-ai/shared-web'
-SAAS_REPO='ssh://git@github.com/determined-ai/saas'
+SAAS_REPO='git@github.com:determined-ai/saas'
```

Comment:
Probably we should add `.git` after `saas` like `saas.git`
https://stackoverflow.com/a/8686727